loafer/ext/aws/providers.py

```diff
@@ -15,6 +16,13 @@ class SQSProvider(AbstractProvider, BaseSQSClient):
     def __init__(self, queue_name, options=None, **kwargs):
         self.queue_name = queue_name
         self._options = options or {}
+        self._backoff_factor = self._options.pop('BackoffFactor', None)
+        if self._backoff_factor:
```

Comment:
eu sei que seria muito estranho, mas se `0` for um backoff factor válido, daí a gente devia explicitamente verificar se não é `None` aqui
```suggestion
        if self._backoff_factor is not None:
```